cropper/app/lib/Crops.scala

```diff
@@ -21,35 +23,53 @@ object Crops {
     val metadata = apiImage.metadata
     val iccColourSpace = FileMetadataHelper.normalisedIccColourSpace(apiImage.fileMetadata)
 
     for {
-      strip <- ImageOperations.cropImage(sourceFile, source.bounds, 100d, Config.tempDir, iccColourSpace, colourModel)
-      file  <- ImageOperations.appendMetadata(strip, metadata)
+      strip <- ImageOperations.cropImage(sourceFile, source.bounds, 100d, Config.tempDir, iccColourSpace, colourModel, mediaType.extension)
+      file: File <- ImageOperations.appendMetadata(strip, metadata)
+
+      //Before apps and frontend can handle PNG24s we need to pngquant PNG24 master crops
+      optimisedFile =  if (colourType == "True Color" || colourType == "True Color with Alpha") {
+
+        val fileName = file.getAbsolutePath()
+
+
+        val optimisedImageName: String = fileName.split('.')(0) + "optimised.png"
+        Seq("pngquant", "--quality", "1-85", fileName, "--output", optimisedImageName).!
+        new File(optimisedImageName)
+      } else file
 
       dimensions  = Dimensions(source.bounds.width, source.bounds.height)
-      filename    = outputFilename(apiImage, source.bounds, dimensions.width, true)
-      sizing      = CropStore.storeCropSizing(file, filename, mediaType, crop, dimensions)
+      filename    = outputFilename(apiImage, source.bounds, dimensions.width, mediaType.extension, true)
+      sizing      = CropStore.storeCropSizing(file, filename, mediaType.name, crop, dimensions)
       dirtyAspect = source.bounds.width.toFloat / source.bounds.height
       aspect      = crop.specification.aspectRatio.flatMap(AspectRatio.clean(_)).getOrElse(dirtyAspect)
+
     }
-    yield MasterCrop(sizing, file, dimensions, aspect)
+    yield MasterCrop(sizing, optimisedFile, dimensions, aspect)
   }
 
-  def createCrops(sourceFile: File, dimensionList: List[Dimensions], apiImage: SourceImage, crop: Crop, mediaType: String): Future[List[Asset]] = {
+  def createCrops(sourceFile: File, dimensionList: List[Dimensions], apiImage: SourceImage, crop: Crop,
+                  mediaType: MimeType): Future[List[Asset]] = {
+
     Future.sequence[Asset, List](dimensionList.map { dimensions =>
       for {
-        file       <- ImageOperations.resizeImage(sourceFile, dimensions, 75d, Config.tempDir)
-        filename    = outputFilename(apiImage, crop.specification.bounds, dimensions.width)
-        sizing     <- CropStore.storeCropSizing(file, filename, mediaType, crop, dimensions)
-        _          <- delete(file)
+        file          <- ImageOperations.resizeImage(sourceFile, dimensions, 75d, Config.tempDir, mediaType.extension)
+        optimisedFile = ImageOperations.optimiseImage(file, mediaType)
+        filename      = outputFilename(apiImage, crop.specification.bounds, dimensions.width, mediaType.extension)
+        sizing       <- CropStore.storeCropSizing(optimisedFile, filename, mediaType.extension, crop, dimensions)
+        _            <- delete(file)
```

Comment:
dont we just need to delete optimisedFile? (cus it just equals file if it's not one of the special png types)